roles/container_runtime/tasks/package_crio.yml

```diff
@@ -32,10 +32,15 @@
 
 - name: Install cri-o
   package:
-    name: "cri-o"
-    state: present
+    name: "{{ crio_pkgs | join(',') }}"
+    state: latest
   register: result
   until: result is succeeded
+  vars:
+    crio_pkgs:
+      - "cri-o"
+      - "cri-tools"
+      - "podman"
```

Comment:
not sure this is going to be available in dockertested @lsm5 @jupierce

Reply:
@sdodson could you PTAL at https://github.com/openshift/aos-cd-jobs/issues/1438 ?

Reply:
At least as far as openshift-ansible CI jobs are concerned I don't know that we need it in dockertested as I'm pushing through work that would enable the OCP 3.x repos for CI jobs which should work for testing stable content. For testing latest pre-release cri-o builds we probably need it in dockertested.

After waiting for a while I've just tagged in a new rhel base image which I believe should unblock this PR